workgroup/src/main/java/software/amazon/athena/workgroup/UpdateHandler.java

```diff
@@ -92,9 +88,25 @@ private UpdateWorkGroupResponse updateWorkgroup(final ResourceModel model) {
       }
 
       // Handle modifications to WorkGroup configuration
-      return clientProxy.injectCredentialsAndInvokeV2(updateWorkGroupRequest, athenaClient::updateWorkGroup);
+      final UpdateWorkGroupRequest.Builder updateRequestBuilder = UpdateWorkGroupRequest.builder()
+          .workGroup(newModel.getName())
+          .description(newModel.getDescription() != null ? newModel.getDescription() : HandlerUtils.DEFAULT_DESCRIPTION)
+          .state(newModel.getState() != null ? newModel.getState() : HandlerUtils.DEFAULT_STATE);
+
+      // Prioritize looking at WorkGroupConfigurationUpdates field
+      if (newModel.getWorkGroupConfiguration() != null) {
+        updateRequestBuilder.configurationUpdates(translator.createSdkConfigurationUpdatesFromCfnConfiguration(newModel.getWorkGroupConfiguration()));
+      }else if (newModel.getWorkGroupConfigurationUpdates() != null) {
```

Comment:
Need extra space after the first } in this line

Reply:
Done.